lib/__tests__/jarSerialization.spec.ts

```diff
@@ -173,12 +175,16 @@ describe('cookieJar serialization', () => {
   describe('with a moderately-sized store', () => {
     let jar: CookieJar
     let expires: Date
+    let start: string
```

Comment:
Let's use fake timers here as well to control creation / lastAccessed timestamps.